.github/workflows/ci.yml

```diff
@@ -22,12 +28,52 @@ jobs:
         run: make check
       - name: distcheck
         run: make distcheck
-  test-ubuntu-oldest:
-    runs-on: ubuntu-16.04
+
+  test-centos6:
+    runs-on: ubuntu-latest
+    container: centos:6
+    steps:
+      - uses: actions/checkout@v1
+      - name: enable special interest group
+        run: yum install centos-release-scl-rh
```

Comment:
Does this enable a 3rd party repository? I'd rather stick with the core repositories. It's OK if Test::Warnings is not installed, it'll just skip the warnings tests.

Reply:
AS long as it is just for testing it is fine. Just makes our life easier.